test/geom/modifier/dodge_test.dart

```diff
@@ -0,0 +1,373 @@
+import 'dart:ui';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:graphic/graphic.dart';
+import 'package:graphic/src/dataflow/tuple.dart';
+import 'package:graphic/src/geom/modifier/dodge.dart';
+
+final _color = Color(0x00000000);
+final _shape = RectShape();
+
+Aes _createAes({required int index, required List<Offset> position}) {
+  return Aes(index: index, position: position, shape: _shape, color: _color);
+}
+
+final epsilon = 0.001;
+
+Matcher _matchesOffsets(List<List<List<Offset>>> expectedOffsets) {
```

Comment:
Needed to create this matcher because `double` operations can cause precision loss and comparing them would cause tests to fail.